src/infrastructure/api/event-router/event-router.test.ts

```diff
@@ -65,4 +65,67 @@ describe('User Routes', function () {
       expect(res.body.events).toHaveLength(2);
     });
   });
+
+  describe('POST /events/edit', function () {
+    it('if the payload is correct it modifies the event', async function () {
+      // Primero, debes crear un evento existente que puedas modificar
+        await request(app)
+        .post('/events/create')
+        .send({
+          codi: 12348173049,
+          denominacio: 'test-event',
+          descripcio: 'test-description',
+          dataIni: new Date(1),
+          dataFi: new Date(2),
+          horari: '2h',
+          adress: 'Passeig de Gràcia',
+          url: 'https://test-url.com',
+        });
+  
+      // Luego, envías una solicitud POST a la ruta /events/edit para modificar el evento
+      const res = await request(app)
+        .post('/events/edit')
+        .send({
+          codi: 12348173049,
+          denominacio: 'new-test-event',
+          descripcio: 'new-test-description',
+        });
+
+  
+      // Verifica que la respuesta de la solicitud sea la esperada y que el evento se haya modificado correctamente
+      expect(res.statusCode).toBe(200);
+      expect(res.body.message).toBe('Evento editado correctamente');
+  
+      //const updatedEvent = await Event.to(12348173049);
+  
+      //expect(res.supdatedEvent.denominacio).toBe('new-test-event');
+      //expect(res.updatedEvent.descripcio).toBe('new-test-description');
+    });
+  
+    it('if the payload is incorrect returns an error', async function () {
+      await request(app)
+        .post('/events/create')
+        .send({
+        codi: 123456789,
+        denominacio: 'test-event',
+        descripcio: 'test-description',
+        dataIni: new Date(1),
+        dataFi: new Date(2),
+        horari: '2h',
+        adress: 'Passeig de Gràcia',
+        url: 'https://test-url.com',
+      });
+  
+      const res = await request(app).post('/events/edit').send({
+        codi: 123456789,
+        denominacio: '',
+      });
+  
+      expect(res.statusCode).toBe(400);
+      expect(res.body.success).toBe(false);
```

Comment:
El codi et peta perk no te un camp de success al body si el payload es incorrecte.
Hauries de comprobar que el camp de errors te coses. per exemple:
```suggestion
      expect(res.body.errors).toBeTruthy()
```